src/main/java/com/salesforce/storm/spout/dynamic/kafka/Consumer.java

```diff
@@ -256,19 +256,12 @@ public void open(
             } else {
                 // We do not have an existing offset saved, so start from the head
                 getKafkaConsumer().seekToBeginning(Collections.singletonList(topicPartition));
-                offset = getKafkaConsumer().position(topicPartition) - 1;
```

Comment:
This throws an exception when we call it on a topic with no offsets.

@Crim Maybe we should catch the position and behave differently rather than drop this altogether. What do you think?